adm_program/system/classes/user.php

```diff
@@ -810,6 +820,7 @@ public function getAllVisibleCategories($categoryType)
                                     WHERE ror_name_intern = \'category_view\'
                                       AND rrd_object_id   = cat_id
                                       AND rrd_rol_id IN ('.replaceValuesArrWithQM($rolIdParams).') )
+                        OR 1 = ' . $flagAdministrator . '
```

Comment:
This is a very ugly hack. Remove the AND clause if flagAdmin is true

Reply:
hmm, code will be more complicated if I remove the AND because there is also a bind parameter. Why we don't use it as it is?

Reply:
I don't think so. If you want i could push my solution

Reply:
Yes you can :)